# ros msg for multiple convex hulls

std_msgs/Header header
plane_seg_ros/convex_hull[] hulls
